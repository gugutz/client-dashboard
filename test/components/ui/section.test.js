import React from 'react'
import renderer from 'react-test-renderer'
import Section from '../../../src/components/ui/section'

describe('<Section /> component', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <Section>
          <div>
Some content
          </div>
        </Section>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
