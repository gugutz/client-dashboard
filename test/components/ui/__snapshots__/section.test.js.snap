// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Section /> component should render correctly 1`] = `
<div
  className="section"
>
  <div>
    Some content
  </div>
</div>
`;
